src/MDMMotionCurve.h

```diff
@@ -119,7 +119,16 @@ typedef NS_ENUM(NSUInteger, MDMBezierMotionCurveDataIndex) {
 typedef NS_ENUM(NSUInteger, MDMSpringMotionCurveDataIndex) {
   MDMSpringMotionCurveDataIndexMass,
   MDMSpringMotionCurveDataIndexTension,
-  MDMSpringMotionCurveDataIndexFriction
+  MDMSpringMotionCurveDataIndexFriction,
+
+  /**
+   The initial velocity of the animation, in units per second.
```

Comment:
What are the units?

Reply:
I struggled to document this one.

The "units" are context-specific, so if the animator is animating a position then it's in screen points/second. If animating rotation, it's radians/second. Do you see a better way to document this?

Reply:
I think your explanation is good. Maybe add it as a non-doc comment if you think it's not really important?

Reply:
Reworked the comments; what do you think?

Reply:
I think it makes sense. I think adding examples like the radians/sec (versus degrees) or points/sec (versus pixels) would help for someone like me. :) Does the client get to pick pixels over points or radians over degrees?

Reply:
The person defining this data is free to define whatever units they want the value to be interpreted as. It's up to the consumer of this data to ensure that they're applying the values correctly to their animation system.

For example, we recommend storing the velocity in terms of "units per second", but Core Animation uses a normalized scalar velocity in terms of the distance traveled, so the MotionAnimator is expected to transform the units/second value into a normalized value.

Alternatively, if someone were to read this interchange into a POP spring, they could set the initial velocity on the POP animation directly (because POP's initial velocity is measured in units/second).